Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -291,6 +291,33 @@ public final class PackageInfoMapper: PackageInfoMapping {
         packageToProject: [String: AbsolutePath],
         swiftToolsVersion: TSCUtility.Version?
     ) throws -> ProjectDescription.Project? {
+        // Hardcoded mapping for some well known libraries, until the logic can handle those properly
+        let productTypes = productTypes.merging(
+            [
+                // Force dynamic frameworks
+                "RxSwift": .framework, // https://github.com/ReactiveX/RxSwift
+            ],
+            uniquingKeysWith: { userDefined, _ in userDefined }
+        )
+        let targetSettings = targetSettings.merging(
```

Comment:
would it make sense to do this if the target is a unit test type?

Reply:
for unit test targets it should not be needed AFAIK
The problem is with frameworks/libraries which depends on XCTest